config/locales/two_factor_authentication/en.yml

```diff
@@ -114,7 +114,7 @@ en:
       sign in</strong>.
     phone_voice_label: Phone number
     piv_cac_fallback:
-      question: Don't have your piv/cac card available?
+      question: Don't have your PIV or CAC available?
```

Comment:
Just to confirm, these strings aren't for IAL2. PIV/CAC verification was removed as of #4819. The updated texts read as an improvement to me all the same, so I think they'd be fine, but wanted to double-check.